bundles/AdminBundle/src/Controller/Admin/ElementController.php

```diff
@@ -339,10 +339,10 @@ public function noteAddAction(Request $request): JsonResponse
     public function findUsagesAction(Request $request): JsonResponse
     {
         $element = null;
-        if ($request->query->get('id')) {
-            $element = Element\Service::getElementById($request->query->get('type'), $request->query->getInt('id'));
-        } elseif ($request->query->get('path')) {
-            $element = Element\Service::getElementByPath($request->query->get('type'), $request->query->get('path'));
+        if ($request->get('id')) {
+            $element = Element\Service::getElementById($request->get('type'), $request->get('id'));
+        } elseif ($request->get('path')) {
+            $element = Element\Service::getElementByPath($request->get('type'), $request->get('path'));
```

Comment:
Apparently, it seems to be the case for many files. Looks to me like the "use my version" conflict resolution strategy was used.